compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java

```diff
@@ -378,10 +402,22 @@ private void defineFunction(DataInputStream dataInStream) throws IOException {
 
         defineMarkDownDocAttachment(invokableSymbol, readDocBytes(dataInStream));
 
+        dataInStream.skip(dataInStream.readLong()); // read and skip method body
+
         scopeToDefine.define(invokableSymbol.name, invokableSymbol);
     }
 
+    private void skipPosition(DataInputStream dataInStream) throws IOException {
+        // TODO find a better way to skip this
```

Comment:
Shall we create an issue to track the Todo also indicating what could be a possible better way so that it can be fixed in the future?